### ISSUE TITLE
See who booked a room

### ISSUE THREAD

**Issue by dueyfinster** [Ericsson/exchangecalendar#77]
Hi,

Enhancement request to see who booked a room by hovering over it in the "Invite Attendees" window:

![seeroomavail](https://cloud.githubusercontent.com/assets/166151/5803220/5e8b18ce-9ff4-11e4-9672-cede8b6da276.png)

Currently the blue bars only show that it's booked, but not who by (so you could ask to use the room).

Cheers,
Neil

**Comment by bavincen** [comment]
Hi, I am not getting Hovering over busy/free information? can you explain please..

**Comment by dueyfinster** [comment]
sure, if you hover over dark blue times (which are busy) in outlook you can see who has booked that time in that room.
![outlook](https://cloud.githubusercontent.com/assets/166151/6186517/3baf0aa4-b372-11e4-92d3-ba25c780c725.png)

**Comment by bavincen** [comment]
Got!  
if i am not wrong is tha room name in second row?

**Comment by dueyfinster** [comment]
Yes it sure is, the room can show who booked it, in this case, Judy.
